demo/actions.py

```diff
@@ -639,3 +642,50 @@ def run(self, dispatcher, tracker, domain) -> List[EventType]:
             dispatcher.utter_template("utter_no_more_steps", tracker)
 
         return []
+
+
+class ActionDocsSearch(Action):
+    def name(self):
+        return "action_docs_search"
+
+    def run(self, dispatcher, tracker, domain):
+        search_text = tracker.latest_message['text']
+        if search_text == "/technical_question{}":
+            last_user_event = dispatcher.get_last_event_for(UserUttered, action_names_to_exclude=None, skip=2)
+            search_text = last_user_event['text']
+
+        # Search of docs pages
+        algolia = AlgoliaAPI(config.algolia_app_id, config.algolia_search_key, config.algolia_docs_index)
+        alg_res = algolia.search(search_text)
+
+        doc_list = algolia.get_algolia_link(alg_res['hits'], 0)
+        doc_list += "\n" + algolia.get_algolia_link(alg_res['hits'], 1) if len(alg_res['hits']) > 1 else ""
+
+        dispatcher.utter_message(
+            "I can't answer your question directly, but I found the following from the docs:\n"
+            + doc_list
+        )
+
+        return []
+
+
+class ActionForumSearch(Action):
+    def name(self):
+        return "action_forum_search"
+
+    def run(self, dispatcher, tracker, domain):
+        search_text = tracker.latest_message["text"]
+        if search_text == "/technical_question{}":
+            last_user_event = dispatcher.get_last_event_for(UserUttered, action_names_to_exclude=None, skip=2)
+            search_text = last_user_event["text"]
+
+        # Search forum
+        discourse = DiscourseAPI("https://forum.rasa.com/search")
+        disc_r = discourse.query(search_text)
```

Comment:
```suggestion
        disc_r = discourse.query(search_text)
```
what does this variable stand for?  please write it out